api/jobs.yaml

```diff
@@ -137,6 +137,71 @@ paths:
           $ref: '#/responses/ServerError'
       tags:
         - Jobs
+  '/jobs/{id}/{task}/attempts':
```

Comment:
TOL Cromwell `call`s are supposed to be called `task`s in JM? There is a concept of `task` in Cromwell already.